### DISCUSSION TITLE
Welcome to EOS Discussions!

### DISCUSSION THREAD

**Discussion by drbacke** [Akkudoktor-EOS/EOS#62] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akrauss** [comment]
Hello Andreas,
first of all, thank you for making the project publicly available. 
I'll make a start here and introduce myself. My name is also Andreas and I'm now far too old for IT ;).
I'm an absolute noob when it comes to programming and prefer to tinker with my scripts rather than program.
I have HomeAssistant running with an almost infinite number of devices, a PV with battery and an electric car at the start. So there's enough data available for testing...

Best regards
Andreas
